src/object/srv_obj_migrate.c

```diff
@@ -1176,9 +1188,42 @@ migrate_one_ult(void *arg)
 		goto out;
 	}
 
-	rc = migrate_dkey(tls, mrone);
-	D_DEBUG(DB_REBUILD, DF_UOID" migrate dkey "DF_KEY" rc %d\n",
-		DP_UOID(mrone->mo_oid), DP_KEY(&mrone->mo_dkey), rc);
+	data_size = daos_iods_len(mrone->mo_iods, mrone->mo_iod_num);
```

Comment:
daos_iods_len() could return -1, it's not well handled in this patch.

Reply:
oh, it is impossible for data_size to be -1 here for rebuild. ok, I will add check here.